docs/server/node/_getConfig.mdx

```diff
@@ -1,5 +1,5 @@
 ```jsx
-const config = Statsig.getConfigSync(user, "awesome_product_details");
```

Comment:
we need something at the TOP of the node sdk about this.  Nothing shows until the bottom